src/facebook/facebook-notifications/facebooknotificationsyncadaptor.cpp

```diff
@@ -64,6 +66,9 @@ void FacebookNotificationSyncAdaptor::finalize(int accountId)
         m_db.purgeOldNotifications(OLD_NOTIFICATION_LIMIT_IN_DAYS);
         m_db.sync();
         m_db.wait();
+
+        // manage image cache
+        purgeCachedImages(&m_imageCacheDb, accountId, QDateTime::currentDateTime().addDays(14 * -1));
```

Comment:
I wonder whether this should be a setting rather than a hardcoded two-week limit.